vehicle/views/vehicle_views.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_vehicle_vehicle_search" model="ir.ui.view">
+        <field name="name">vehicle.vehicle.search</field>
+        <field name="model">vehicle.vehicle</field>
+        <field name="arch" type="xml">
+            <search string="Vehicle Search">
+                <filter
+                    string="Archived"
+                    name="inactive"
+                    domain="[('active','=',False)]"
+                />
+                <separator />
+                <field name="name" filter_domain="[('name', 'ilike', self)]" />
+            </search>
+        </field>
+    </record>
+    <record id="view_vehicle_vehicle_form" model="ir.ui.view">
+        <field name="name">vehicle.vehicle.form</field>
+        <field name="model">vehicle.vehicle</field>
+        <field name="arch" type="xml">
+            <form string="Vehicle">
+                <header>
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="toggle_active"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-check"
+                        >
+                            <field name="active"/>
```

Comment:
This button looks strange:

![imagen](https://user-images.githubusercontent.com/12749832/124641764-56a0cb00-de8f-11eb-912d-8518c93f5bb7.png)

Reply:
I was looking for the bug in this field but apparently in the helpdesk_ticket addon in the stages form this visualization is also used and the same thing happens to it. We can add the field with a basic visualization in the form, what do you think?

Reply:
Please take a look at https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0 :

![imagen](https://user-images.githubusercontent.com/12749832/124730407-99a38280-df11-11eb-924a-6f807b9ba2f2.png)